hrjobcontract/CRM/Hrjobcontract/Test/Fabricator/BaseAPIFabricator.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+abstract class CRM_Hrjobcontract_Test_Fabricator_BaseAPIFabricator {
```

Comment:
Do we need the `_Test_` bit in the name of the classes?
`CRM_ExtensionName_Fabricator_EntityName` is not good enough?